cmd/rhoas/main.go

```diff
@@ -58,12 +58,9 @@ func main() {
 		}
 		return
 	}
-
-	if err != nil {
-		logger.Error(wrapErrorf(err, localizer))
-		build.CheckForUpdate(context.Background(), logger, localizer)
-		os.Exit(1)
-	}
+	logger.Error(wrapErrorf(err, localizer))
```

Comment:
FIxing logic here. It should be the opposite

Reply:
Actually I needed to do it this way because of linting issue. 
So logic is as follows:

if not error and debug check update
otherwise (error) print error and check update 

it is confusing but os.Exit(1) typically should be the last statement (there is no return after that ;) :)